src/Service.Tests/Unittests/RestServiceUnitTests.cs

```diff
@@ -40,7 +40,7 @@ public class RestServiceUnitTests
         /// <param name="expectedPrimaryKeyRoute">The primary key route we
         /// expect to parse from route.</param>
         [DataTestMethod]
-        [DataRow("rest-api/Book/id/1", "/rest-api", "Book", "id/1")]
+        [DataRow("/rest-api/Book/id/1", "/rest-api", "Book", "id/1")]
```

Comment:
see comment on other test's modification: this should be considered an invalid route.

Reply:
added back. Created [this](https://github.com/Azure/data-api-builder/issues/1640) issue for discussions related to consecutive slashes.